src/lib/questions.ts

```diff
@@ -496,6 +585,8 @@ export const questionsAndText: (
 				message: "Do you prefer tab or space indentation?",
 				initial: "Tab",
 				choices: ["Tab", "Space (4)"],
+				import: async (ctx) =>
+					(await ctx.analyzeCode("\t", "  ")) ? "Tab" : "Space (4)",
```

Comment:
If ESLint is used, we could look at the indent or @typescript-eslint/indent rules instead.

Reply:
Also, shouldn't that be 4 spaces instead?

Reply:
First comment: yes, but in any case this comparison will work - parsing rules is a bit more tricky and might change over time.
Second comment: true, but I prefer 2 spaces, so we kind of "upgrade" from 2 spaces to 4 but keep spaces. Two spaces will also match four spaces ;-).